ui/src/app/routing/guards.tsx

```diff
@@ -29,7 +29,7 @@ export const signInGuard: Guard = {
   redirectPath: '/login',
 };
 
-const userIsEnabled = (userDisabledInDb: boolean) =>
+export const userIsEnabled = (userDisabledInDb: boolean) =>
```

Comment:
I know it's an existing issue, but we can be more precise with this.  How about `shouldShowDisabledPage()` ?  (and flip the logic accordingly)

Reply:
I felt userIsDisbaled is making more sense when we are reading guard section, let me know if you think otherwise